twilio/endpoint.bal

```diff
@@ -18,15 +18,23 @@ import ballerina/auth;
 import ballerina/http;
 import ballerina/mime;
 
-# Client for Twilio endpoint.
+# The Twilio API provides capability to access its platform for communications. These APIs connects 
+# the software layer and communications networks around the world to allow users to call and message anyone, globally. 
 #
 # + accountSId - Unique identifier of the account
 # + basicClient - HTTP client endpoint for basic api
 @display {label: "Twilio Client", iconPath: "logo.svg"}
-public client class Client {
-    public string accountSId;
-    public http:Client basicClient;
+public isolated client class Client {
+    private final string accountSId;
+    private final http:Client basicClient;
 
+    # Gets invoked to initialize the `connector`.
```

Comment:
What gets invoked to initialize the connector? Can we have this information in the following format?

"To initialize the connector, invoke the XXX and set the API credentials."